src/web/js/trove/world-lib.js

```diff
@@ -650,13 +602,9 @@
       }
     };
 
-
-
-
-
     //////////////////////////////////////////////////////////////////////
 
-    var bigBang, StopWhenHandler;
+    var bigBang, StopWhenHandler, LastImageHandler;
```

Comment:
Is this the same as or different from your last-image-handler PR?

Reply:
Same.